cpp/src/aztec/honk/composer/composer_helper/composer_helper.cpp

```diff
@@ -293,14 +300,18 @@ UnrolledVerifier ComposerHelper<CircuitConstructor>::create_unrolled_verifier(Ci
 }
 
 template <typename CircuitConstructor>
-UnrolledProver ComposerHelper<CircuitConstructor>::create_unrolled_prover(CircuitConstructor& circuit_constructor)
+template <typename Flavor>
+// TODO(Cody): this file should be generic with regard to flavor/arithmetization/whatever.
+StandardUnrolledProver ComposerHelper<CircuitConstructor>::create_unrolled_prover(
```

Comment:
Yeah unrolled is a terrible name. Plan is to kill the linearization trick, then I think we should just have `create_prover`. But for now we need both, even here, because `ComposerHelper` is generic and needs to hold stuff for Plonk.